src/cs/implementations/setup.rs

```diff
@@ -185,12 +188,12 @@ impl<
             .max()
             .unwrap_or(0);
 
-        dbg!(required_size);
-        dbg!(max_copiable_in_specialized_columns);
-        dbg!(max_witnesses_in_general_purpose_columns);
-        dbg!(max_witnesses_in_specialized_columns);
-        dbg!(max_constants_for_general_purpose_gates);
-        dbg!(max_in_column_for_specialized_gates);
+        log!("required size = {}", required_size);
+        log!("{}", max_copiable_in_specialized_columns);
```

Comment:
Probably better to put `log!("max_copiable_in_specialized_columns = {}", max_copiable_in_specialized_columns)` for readability